dns.c

```diff
@@ -22,10 +22,12 @@ extern int res_search();
 static unsigned short getshort(c) unsigned char *c;
 { unsigned short u; u = c[0]; return (u << 8) + c[1]; }
 
-static union { HEADER hdr; unsigned char buf[PACKETSZ]; } response;
+static struct { unsigned char *buf; } response;
+static int responsebuflen = 0;
 static int responselen;
 static unsigned char *responseend;
 static unsigned char *responsepos;
+static u_long saveresoptions;
```

Comment:
plain unsigned long, I wonder why this compiles at all

Reply:
Maybe it doesn’t? This was an extremely low-effort initial submission on my part, just seeing the branch already exists and creating a PR. If it’s probably agreeable for 1.09 then I’ll make more effort, starting now :-)

Reply:
Comes from `<sys/types.h>`, at least on macOS

Reply:
Fixed in #201.